pages/index.js

```diff
@@ -1,14 +1,29 @@
 import Head from 'next/head'
 import Footer from '../components/Footer'
 import Navbar from '../components/Navbar'
-import styles from '../styles/Home.module.css'
+import styles from '../styles/Home.module.css'  
+import Herosection from './herosection'
+import RecMovies from '../components/RecMovies'
+import Link from 'next/link'
 
 export default function Home() {
   return (
-    <div>
-      <h1>homepage</h1>
+  <div className="HomePage">
+    <div className="ImageSlider">
+      <Herosection/>
+    </div>
+    <div className="SelectedMovies">
+      <RecMovies /> 
+    </div>
+    <div className={styles.About}>
+      <h1>About us</h1>
       <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
-      <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+      <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>  
+      <Link href = "about"><a>read more..</a></Link>
+    </div>
+    <div className={styles.quote}>
+      <h1>"No man can win every battle, but no man should fall without a struggle."</h1>
```

Comment:
Line 25 - To avoid error in the editor for this line. A quick fix is to replace the " symbols with &quot;

Reply:
there is never any need to use &quot; in element content in HTML (XHTML or any other HTML version). There is nothing in any HTML specification that would assign any special meaning to the plain character " there.